services/demo/webpack.config.js

```diff
@@ -8,7 +8,7 @@ const packagePath = (relPath) => path.resolve(__dirname, relPath);
 const rootPath = (relPath) => path.resolve(__dirname, '../..', relPath);
 
 module.exports = {
-    devtool: 'inline-source-map',
+    devtool: 'cheap-module-eval-source-map',
```

Comment:
couldn't find source code in the chrome debugger with the original source map settings

Reply:
Thanks for fixing this!